reference/api/tasks.md

```diff
@@ -2,10 +2,188 @@
 
 The `/tasks` route gives information about the progress of [asynchronous operations](/learn/advanced/asynchronous_operations.md).
 
+## Task object
+
+```json
+{
+  "uid": 4,
+  "indexUid" :"movie",
+  "status": "failed",
+  "type": "indexDeletion",
+  "details": {
+    "deletedDocuments": 0
+  },
+    "error": {
+      "message": "Index `movie` not found.",
+      "code": "index_not_found",
+      "type": "invalid_request",
+      "link": "https://docs.meilisearch.com/errors#index_not_found"
+    },
+      "duration": "PT0.001192S",
+      "enqueuedAt": "2022-08-04T12:28:15.159167Z",
+      "startedAt": "2022-08-04T12:28:15.161996Z",
+      "finishedAt": "2022-08-04T12:28:15.163188Z"
+}
+```
+
+### `uid`
+
+**Type**: Integer
+
+**Description**: Unique sequential identifier of the task
+
 :::note
 The task `uid` is incremented **globally.**
 :::
 
+### `indexUid`
+
+**Type**: String
+
+**Description**: Unique identifier of the index modified by the task
+
+::: note
+This value is always `null` for `dumpCreation` tasks.
+:::
+
+### `status`
+
+**Type**: String
+
+**Description**: Status of the task. Possible values are `enqueued`, `processing`, `succeeded`, `failed`
+
+### `type`
+
+**Type**: String
+
+**Description**: Type of operation performed by the task. Possible values are `indexCreation`, `indexUpdate`, `indexDeletion`, `documentAdditionOrUpdate`, `documentDeletion`, `settingsUpdate`, `dumpCreation`
+
+### `details`
+
+**Type**: Object
+
+**Description**: Detailed information on the task payload. This object's contents depend on the task's `type`
+
+::: note
+The `details` object will only show values for the updated field. For example, if you update the faceting settings, the `details` object will only contain the updated faceting value:
+
+```json
+{
+   "uid": 3,
+   "indexUid": "movies",
+   "status": "succeeded",
+   "type": "settingsUpdate",
+   "details": {
+      "faceting": {
+         "maxValuesPerFacet": 2
+      }
+   },
+   "duration": "PT0.000682S",
+   "enqueuedAt": "2022-08-02T15:41:57.955977Z",
+   "startedAt": "2022-08-02T15:41:57.957002Z",
+   "finishedAt": "2022-08-02T15:41:57.957684Z"
+}
+```
+
+:::
+
+#### `documentAdditionOrUpdate`
+
+<style>
+table th:first-of-type {
+    width: 10%;
```

Comment:
I don't completely understand how this works. When I set it to 10%, it works for every table on the docs. If I change it, it only seems to work on a few tables in `tasks.md`

Reply:
I do not understand this either. In any case, since it appears to be affecting every table in the docs—and as far as I can tell, is doing exactly what we wanted: making every table use the full `content-width`—it would be better if it was in a more explicit place rather than this particular file. What happens if you add this to `.vuepress/styles/index.styl`, for example?

Reply:
Moved to `.vuepress/styles/index.styl`. I set `width` to `10`. All tables are the same width. 

If I set it to `25`, tables with 3 columns look better, but somehow all tables have different widths. We can work on this in a seperate PR

`width: 10%;`:
<img width="774" alt="Screen Shot 2022-08-09 at 15 27 01" src="https://user-images.githubusercontent.com/90181761/183641246-eb4daf14-edf5-4186-a426-b367ef04b311.png">

`width: 25%;`:
<img width="712" alt="Screen Shot 2022-08-09 at 15 45 16" src="https://user-images.githubusercontent.com/90181761/183641269-1c4093ee-90a2-4f21-8348-685c20577435.png">

Reply:
I think working on a separate PR is a good idea. I think the rest of these changes are good and important, so wouldn't want to delay it because of presentation tweaks.